content/docs/guide-development.md

```diff
@@ -117,7 +117,60 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
+* Create dedicated annotations calls to `astroquery` functions and add those to the project Knowledge Graph (`renku-aqs-annotation` plugin)
+
+Those two set of features are provided within different plugins.
+
+### Visualizing project Knowledge Graph with `renku-graph-vis` plugin
+
+Starting from the knowledge graph, this is used to generate an interactive graphical representation of the graph itself.
+
+The plugin provides two CLI commands:
+  * `display` to generate a representation of the graph over an output image
+  * `show-graph` to start an interactive visualization of the graph over the browser
+
+In addition, the plugin offers a dynamic visualization of the graph during an interactive renku session.
+
+The primary benefit introduced is the ability to have a live overview of the ongoing development within an interactive Renku session. This can be seen within the animation below, where the graph is automatically updated with information about the execution of a notebook upon its completion.
+
+![](renkulab_execution_example_2.gif)
+
+This visualization also includes the ODA ontology, providing valuable insights into the types of entities within it that are known to the ontology, and therefore helping during the workflow development. The image below displays a graph where the ODA ontology has been imported, and it can be seen that the `SimbadClass` node is an instance of the `AstroqueryModule` class, while `Mrk 421` is an instance of the `AstrophysicalObject` class.
+
+![](details_astroquery_annotations_2.png)
+
+More technical details are presented in the README of the repo page: [https://github.com/oda-hub/renku-graph-vis/](https://github.com/oda-hub/renku-graph-vis/)
+
+#### Plugin Installation
+
+The plugin can be installed either via pip:
+
+```bash
+pip install renku_graph_vis
+```
+
+Or can be made available within a Renku session, by adding those in the list of requirements of the Renku project, within your `requirements.txt` file.
```

Comment:
"by adding those" - who are these those?

Reply:
remained from previous "version" of this paragraph